backend/server/api.go

```diff
@@ -152,6 +153,16 @@ func bindAPIRoutes(r *mux.Router, db *database.Connection, contentStore contents
 		return services.ListTagsForOperation(r.Context(), db, i)
 	}))
 
+	route(r, "GET", "/api/operations/{operation_slug}/evidence", jsonHandler(func(r *http.Request) (interface{}, error) {
+		dr := dissectJSONRequest(r)
+		tlq, _ := helpers.ParseTimelineQuery(dr.FromQuery("q").AsString())
```

Comment:
A couple of points here:

1. You should handle the error.  This is user input, so it's reasonable to expect that the data would be unparsable. Just returning the error would be fine here.
2. Although `q` is pretty common as a name, we're currently using `query` for the same function in web. We should probably have consistent naming across the two rest endpoints

Reply:
```suggestion
		tlq, err := helpers.ParseTimelineQuery(dr.FromQuery("query").AsString())
		if err != nil {
			return nil, err
		}
```